src/main/java/at/hannibal2/skyhanni/features/garden/FarmingMilestoneCommand.kt

```diff
@@ -62,7 +94,7 @@ object FarmingMilestoneCommand {
         } else listOf()
     }
 
-    private fun getValidNumber(entry: String?) = entry?.toIntOrNull()?.coerceIn(0, 46)
+    private fun getValidNumber(entry: String?) = entry?.toIntOrNull()
```

Comment:
do we still need this funtion?

Reply:
Doesn't seem so, I'll just replace it with direct calls.